.taskcluster.yml

```diff
@@ -134,32 +134,54 @@ tasks:
                 description: Schedules the build and test tasks for Application Services.
         "tasks_for == 'github-release' && event['action'] == 'published'":
           $let:
-            beetmover_worker_type: appsv-beetmover-v1
-            beetmover_bucket: maven-production
-            beetmover_bucket_public_url: https://maven.mozilla.org/
-            tag: ${event.release.tag_name}
-            release_task_definition:
-              payload:
-                features:
-                  chainOfTrust: true
-              scopes:
-                # So that we can publish on Maven using beetmover
-                - project:mozilla:application-services:releng:beetmover:action:push-to-maven
-                # So that we can upload symbols to Socorro
-                - "secrets:get:project/application-services/symbols-token"
+            is_staging: 'event.repository.html_url == "https://github.com/mozilla/application-services"'
```

Comment:
Hm, I'm confused a bit. If there's a github release published and we're within the trusted `application-services` repository, we set this to staging? I would've assumed the opposite with `!=` 🤔

Reply:
Wait, yeah, :thinking: good point

Reply:
On the other hand, your staging release worked like a charm. I'm a bit confused to be honest 🤔

Reply:
I'm guessing I had the repo set to `mitchhentges/application-services` when I did the staging run :)

Reply:
Should we trigger another staging release then to confirm?

Reply:
Good point, it's running [here](https://tools.taskcluster.net/task-group-inspector/#/a3RUIGGOT4KpHq0JDqmwqA)